Update React Quickstart to mention scopes on the API (#9559)
auth0/docs: articles/quickstart/spa/react/02-calling-an-api.md

);
```

:::note

> As Auth0 is only able to issue Tokens for custom scopes that exist on your API, ensure the scopes used above are defined when [setting up an API](https://auth0.com/docs/getting-started/set-up-api) with Auth0.

Reviewer: ```suggestion
As Auth0 is only able to issue tokens for custom scopes that exist on your API, ensure the scopes used above are defined when [setting up an API](https://auth0.com/docs/getting-started/set-up-api) with Auth0.
```

> As Auth0 is only able to issue tokens for custom scopes that exist on your API, ensure the scopes used above are defined when [setting up an API](https://auth0.com/docs/getting-started/set-up-api) with Auth0.

Reviewer 2: ```suggestion
As Auth0 can only issue tokens for custom scopes that exist on your API, ensure that you define the scopes used above when [setting up an API](https://auth0.com/docs/getting-started/set-up-api) with Auth0.
```

I would reword it as the above using the active voice :)
